src/esp/physics/PhysicsManager.h

```diff
@@ -227,9 +230,10 @@ class PhysicsManager {
 
   /** @brief Set the friction coefficient of the scene collision geometry. See
    * @ref sceneNode_.
-   * @param The scalar friction coefficient of the scene geometry.
+   * @param frictionCoefficient The scalar friction coefficient of the scene
+   * geometry.
    */
-  virtual void setSceneFrictionCoefficient(const double){};
+  virtual void setSceneFrictionCoefficient(const double frictionCoefficient){};
```

Comment:
This will end up bringing an unused parameter warning back.  On way to deal wit this is to not have the definition inline here.  Just do `virtual void setSceneFrictionCoefficient(const double frictionCoefficient);` then have `void PhysicsManager::setSceneFrictionCoefficient(const double){};` in the `.cpp`

Reply:
Thanks, Erik. This works well.

Reply:
Or put the [CORRADE_UNUSED](https://doc.magnum.graphics/corrade/Macros_8h.html#aaf99b187cce2ff1eadddf1507ace2d69) on it :)

Reply:
Oooooo.  That seems even better :)